bench-vortex/src/bin/clickbench.rs

```diff
@@ -346,13 +366,20 @@ fn init_data_source(
     engine_ctx: &EngineCtx,
     tokio_runtime: &tokio::runtime::Runtime,
 ) -> anyhow::Result<()> {
+    let dataset = BenchmarkDataset::ClickBench { single_file };
```

Comment:
nit: let's just introduce this at the top once

Reply:
sure